Source/AddOns/DynamicRaceDetection/ThreadMonitor/Monitoring/Threads/ThreadExecutionMonitorDispatcher.cs

```diff
@@ -204,14 +206,23 @@ private static string TryGetClassName(UIntPtr location, UIntPtr objH)
         private string GetDebugInformation(UIntPtr location, UIntPtr objH)
         {
             string sourceInfo = "";
+            List<string> debugInfo = new List<string>();
             if (Configuration.EnableReadWriteTracing)
             {
+                for (int i = 4; i < 15; i++)
```

Comment:
Why start at level 4?

Reply:
stackframe 0-4 actually contains call stack information of the Dynamic Race detection code, which is not required.